File: py/ege18.47.py
f = open("ege18.47.txt")

e = [int(l) for l in f if l != '\n']

k = 0
apx = [] 
n = 0

for x in e:

    if n < 11:
        n += 1
        continue

    apx.append(e[n - 11])

    for px in apx:
        if px + x < 200:
            k += 1

    n += 1

print(k)

File: c/ege18.47.c
#include <stdio.h>

#define N 1024

int main() {
    int a[N];
    int n = 0;
    int k = 0;
    int b = 0;

    FILE *f = fopen("ege18.47.txt", "r");
    int x;
    while(fscanf(f, "%d", &x) != EOF) { 
//        printf("%d\n", x);
        a[n] = x;
        n++;
    }

    for(int c = 11; c < 1000; c++) {
        b = 0;
        while(b <= c - 11) {
            if(a[c] + a[b] < 200) {
                k++;
            }
            b++;
        }
    }
        
    printf("%d\n", k);
}
